app/assets/stylesheets/govuk_publishing_components/components/_feedback.scss

```diff
@@ -224,12 +227,30 @@
     }
   }
 
+  @include govuk-media-query($until: tablet) {
```

Comment:
not sure these styles are doing anything? at least i'm not seeing a change on mobile when i remove them